### PR TITLE
feature/cyber_ark_audit_logs_add_event_fields

### PR DESCRIPTION
**Pull request by clement-burtscher-sekoia**
## Description

Relates to https://github.com/SekoiaLab/integration/issues/1900

### Added

- Parse new ECS fields for ITDR events (`event.dataset: ITDR`, `event.action: New alert created`):
  - `event.outcome`
  - `event.severity`
  - `event.url`
- Parse new custom fields for ITDR events (`event.dataset: ITDR`, `event.action: New alert created`):
  - `cyberark.audit.status`
- Add anonymized ITDR test fixtures from raw CSV logs

### Fixed

- Normalize ITDR `event.outcome` from `customData.Status` (only when it is present) with ECS semantics: `success`/`failure` when resolvable, `unknown` otherwise

## Type of Change

- [ ] New intake format
- [x] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product: CyberArk/cyberark-audit-logs

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Extend CyberArk audit log parsing to support ITDR alert metadata and severity-aware event enrichment.

New Features:
- Add ECS and CyberArk-specific fields for ITDR alert events, including alert links, normalized outcomes, severity levels, and raw statuses.

Bug Fixes:
- Normalize ITDR alert outcomes to the ECS `unknown` value while retaining the original status.

Tests:
- Add extensive anonymized ITDR fixtures covering critical, high, medium, and low severity alerts.

### CODE REVIEW

File: CyberArk/cyberark-audit-logs/ingest/parser.yml [hunk 1]
```diff
@@ -114,6 +114,27 @@ stages:
 
   set_specific_fields:
     actions:
+      - set:
+          event.url: "{{parsed_event.message.customData.Alert_link}}"
+          event.outcome: "{{parsed_event.message.customData.Status}}"
+        filter: "{{parsed_event.message.applicationCode == 'ITDR'}}"
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply CyberArk/cyberark-audit-logs/ingest/parser.yml:121]: Fixed with commit [Apply changes requested by Copilot review](https://github.com/SEKOIA-IO/intake-formats/pull/2856/changes/e60463f7ffb50c46f1ea5217b73c4598de8e7a2f)

File: CyberArk/cyberark-audit-logs/ingest/parser.yml [hunk 2]
```diff
@@ -114,6 +114,42 @@ stages:
 
   set_specific_fields:
     actions:
+      - set:
+          event.url: "{{parsed_event.message.customData.Alert_link}}"
+        filter: "{{parsed_event.message.applicationCode == 'ITDR'}}"
+
+      - set:
+          cyberark.audit.status: "{{parsed_event.message.customData.Status}}"
+        filter: "{{parsed_event.message.applicationCode == 'ITDR' and parsed_event.message.customData.Status | length > 0}}"
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply CyberArk/cyberark-audit-logs/ingest/parser.yml]: Fixed with commit [Apply changes requested by copilot review](https://github.com/SEKOIA-IO/intake-formats/pull/2856/changes/630764afbb5dcda8308b00193ce6c413362d91ec)

**Review by TOUFIKIzakarya — APPROVED** [review]
LGFM
